.github/workflows/snakemake-dry-run.yaml

```diff
@@ -2,15 +2,15 @@ name: Snakemake dry run
 
 # Controls when the action will run.
 on:
-   # Triggers the workflow on push or pull request events but only for the master and develop branch
+   # Triggers the workflow on push or pull request events but only for the main and develop branch
   push:
     branches:
      - develop
-     - master
+     - main
```

Comment:
should we rename main to master so that all repositories have the same branches?